Library/Homebrew/cmd/audit.rb

```diff
@@ -182,6 +186,10 @@ def audit_file
       problem "'__END__' was found, but 'DATA' is not used"
     end
 
+    if text =~ /inreplace .* do/ && text.has_one_sub?
```

Comment:
This would cause many false positives if the `gsub` is not in the `inreplace` block. For example,
```
inreplace "foo" do |s|
  s.change_make_var! ...
end
...
bar.gsub!
```

At the same time, it will fail to catch when multiple `inreplace` exist. 

I suggest use a multi-line regex, such as:(not tested yet)
```
/inreplace .* do .*$\n.*\.g?sub!?.*$\n.*end/m
```